### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by GDGiantDwarf**
README was made more user friendly and more themed around Allfeat

### CODE REVIEW

**Comment by GDGiantDwarf** [comment]
Keep in mind that this Repo contains the tools to build a website that hosts the documentation. It is not really meant to be used locally ; MkDocs will build the HTML+CSS for a nice looking documentation.

If the documentation is to be shared, it seems more appropriate to share a link to a server that hosts it, rather than sharing the repo that builds it, hence why editing README seems like an odd issue to have

**Comment by SailorSnoW** [comment]
Yes absolutely. Sorry if it was not clear enough, but the README should be here to help contributors to test their changes in a local developement environment, that's why it still important to have a README explaining how to setup it and how to contribute to the documentation.

Maybe you could remove all the parts explaining what is Allfeat in the README u made and instead having a more important part on how to contribute to the documentation (like explaining the good path -> create an issue for a missing doc/a bug/an incorrect text then create a PR...) maybe we should get inspiration on other documentation repos of other open-source projects

File: README.md [hunk 1]
```diff
@@ -1,38 +1,69 @@
-# This is a MkDocs project
+# Welcome to Allfeat Documentation! 🚀
 
-## Install
+Welcome to the Allfeat documentation! This guide is your gateway to understanding and leveraging Allfeat's cutting-edge platform for decentralized applications (DApps) and smart contracts. Whether you're a developer, validator, or simply curious about the possibilities of decentralized music industry infrastructure, you've come to the right place! 💿🎶
 
-### Prerequisites
+## What is Allfeat? 🎵
```
Comment (SailorSnoW) [inline README.md]: This part isn't required to appear in a README I guess, Allfeat explanation is already in the doc

File: README.md [hunk 2]
```diff
@@ -1,38 +1,69 @@
...
+### Start a Local Server
+
+Spin up a local server to preview your Allfeat documentation.
+
 ```bash
 mkdocs serve
 ```
 
-## Building your site
+### Build Your Site
```
Comment (SailorSnoW) [inline README.md:55]: not really required if we only aim this repo to be used in a local environment for contributors

**Review by SailorSnoW — CHANGES_REQUESTED** [review]

**Comment by SailorSnoW** [comment]
As contributing is already in the doc, yeah maybe is redundant to put it again the README my bad.

File: README.md (hunk 2 above)
Comment (GDGiantDwarf) [reply README.md:55]: the main aim is for it to be hosted by Allfeat. Contributors will then make a pull request if necessary, possibly test locally how the build looks before sending the pull request.

**Comment by GDGiantDwarf** [comment]
I've finalized this PR, so that the readme still at least looks nicer, and has some amount of introduction for contributors making pull requests

**Review by SailorSnoW — APPROVED** [review]

**Pull request merged**